Decisions.TruCap/Steps/DocumentMonitorSteps.cs

```diff
@@ -27,21 +27,23 @@ public class DocumentMonitorSteps
             }
             
             string baseUrl = ModuleSettingsAccessor<TruCapSettings>.GetSettings().GetBaseDocumentMonitorUrl(overrideBaseUrl);
-            Task<HttpResponseMessage> response = TruCapRest.TruCapGet(
+            HttpResponseMessage response = TruCapRest.TruCapGet(
                 $"{baseUrl}/Project/{project}/DocumentSubType/{docSubType}/FromDate/{startDate}/ToDate/{endDate}",
                 authentication);
 
             try
             {
-                return DocumentMonitorResponse.JsonDeserialize(await response.Result.Content.ReadAsStringAsync());
+                Task<string> resultTask = response.Content.ReadAsStringAsync();
+                resultTask.Wait();
+                return DocumentMonitorResponse.JsonDeserialize(resultTask.Result);
             }
             catch (Exception ex)
             {
-                throw new Exception(await response.Result.Content.ReadAsStringAsync());
+                throw new Exception(response.Content.ReadAsStringAsync().Result);
```

Comment:
If we're throwing an exception we should throw a `BusinessRuleException` containing the details of the exception. This is the response and if we were going to keep it this way we'd have to wait on the task.